load-tests/query-operation/src/query-operation.bal

```diff
@@ -0,0 +1,91 @@
+import ballerinax/mssql.driver as _;
+import ballerina/http;
+
+// The username , password and name of the MSSQL database
+configurable string dbHost = ?;
+configurable string dbUsername = ?;
+configurable string dbPassword = ?;
+configurable string dbName = ?;
+configurable int dbPort = ?;
+
+type Customer record {
+    int customerId;
+    string lastName;
+    string firstName;
+    int registrationId;
+    float creditLimit;
+    string country;
+};
+
+final mssql:Client dbClient = check new(host = dbHost, user = dbUsername, password = dbPassword, port = dbPort, database = dbName);
+
+public function main() returns error? {
+    _ = check dbClient->execute(`
+        CREATE TABLE Customers (
+            customerId INT NOT NULL IDENTITY PRIMARY KEY,
+            firstName VARCHAR(300),
+            lastName  VARCHAR(300),
+            registrationID INT,
+            creditLimit FLOAT,
+            country  VARCHAR(300)
+        );
+    `);
+
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Peter','Stuart', 1, 5000.75, 'USA')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Dan', 'Brown', 2, 10000, 'UK')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Peter','Stuart', 3, 5000.75, 'USA')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Dan', 'Brown', 4, 10000, 'UK')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Peter','Stuart', 5, 5000.75, 'USA')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Dan', 'Brown', 6, 10000, 'UK')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Peter','Stuart', 7, 5000.75, 'USA')`);
+    _ = check dbClient->execute(`
+        INSERT INTO Customers(firstName, lastName, registrationID, creditLimit, country)
+        VALUES ('Dan', 'Brown', 8, 10000, 'UK')`);
+}
+
+isolated service /customer on new http:Listener(9092) {
+    resource isolated function get .(http:Caller caller, int id) returns error? {
+        Customer|error customer = dbClient->queryRow(`SELECT * FROM Customers WHERE customerId = ${id}`);
+
+        http:Response response = new;
+        if customer is error {
+            response.statusCode = 500;
+            response.setPayload(customer.toString());
+        } else {
+            response.statusCode = 200;
+            response.setPayload(customer.toString());
+        }
+        _ = check caller->respond(response);
+    }
```

Comment:
```suggestion
    resource isolated function get .(int id) returns string|error {
        Customer customer = check dbClient->queryRow(`SELECT * FROM Customers WHERE customerId = ${id}`);
        return customer.toString();
    }
```

Reply:
shall we simplify the code like below?
This should do the same thing

Reply:
Fixed in b65dcf1